apps/ledger-live-desktop/src/renderer/families/tron/Votes/index.tsx

```diff
@@ -103,8 +101,8 @@ const Delegation = ({ account }: { account: TronAccount }) => {
     });
   };
 
-  const claimOnClick = ({ liveAppId }: { liveAppId: string }) => {
-    const value = `/platform/${liveAppId}`;
+  const claimOnClick = () => {
+    const value = "/platform/stakekit";
```

Comment:
@sprohaszka-ledger I think it's because : "Remove the noop conditional in the claimOnClick button as the disabled property already handles that." (see description), you confirm @Philippoes ?

Reply:
Yes. See https://github.com/LedgerHQ/ledger-live/blob/df1dcbffe901d7c4baddb46a06b08a4ed5b7a17e/apps/ledger-live-desktop/src/renderer/components/Button.tsx#L385

I tested it and the click doesn't do anything if the button is disabled.

Reply:
@sprohaszka-ledger disabled attribute covers this